test/test_files/copy/copy_partial_column.test

```diff
@@ -52,7 +52,7 @@ matthew perry|2012-12-11 20:07:22||1950-05-14|20 years 30 days 48:00:00|[fwh9y81
 Binder exception: Number of columns mismatch. Expected 7 but got 9.
 -STATEMENT COPY knows3 (meetTime, date, validInterval, comments, summary, dummy) FROM "${KUZU_ROOT_DIRECTORY}/dataset/tinysnb/eKnows.csv" ;
 ---- error
-Binder exception: Table knows3 does not contain column dummy.
+Binder exception: Table knows3_person_person does not contain column dummy.
```

Comment:
I think we should fix these printings before merging the PR. There are two options come into my mind

- Propagate user input to error message (it should certainly work in this case)
- Add a printableName to RelCatalogEntry

I would go with option one first. It's more work but also more natural.

Reply:
Fixed now.